src/Resources/views/CRUD/base_list.html.twig

```diff
@@ -182,68 +182,82 @@ file that was distributed with this source code.
                                                 {% endblock %}
                                             </script>
 
-                                        {% block batch_actions %}
```

Comment:
Try to not change indentations please (even though I agree with all your changes, it will keep the PR as simple as possible)

Reply:
Hey @jordisala1991 

I undestand what you mean but it's quite hard to keep the same indentation, because the IDE auto ident the code, and I'm changing too much in many places :/


But I tried to fix again! :)